projects/ids-enterprise-ng/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "ids-enterprise-ng",
   "slug": "ids-enterprise-ng",
-  "version": "5.3.0-dev",
+  "version": "5.1.0-dev.20181219",
   "dependencies": {
     "@types/jquery": "3.3.24",
     "@types/d3": "4.13.0",
-    "ids-enterprise": "4.18.0-dev.20190409",
+    "ids-enterprise": "4.19.0-dev.20190424",
     "jquery": "3.3.1",
     "d3": "4.13.0"
   },
   "peerDependencies": {
-    "@angular/common": "^7.2.0",
-    "@angular/core": "^7.2.0"
+    "@angular/common": "^7.1.0",
```

Comment:
This seems strange to me as you moved it back? But actually its 7.2.9 now in the main package.json https://github.com/infor-design/enterprise-ng/blob/master/package.json#L31

Reply:
I've moved this back to 7.2 do we want to move it to 7.2.9?